server/modules/survey/service/publish/jobs/recordsRemovedCyclesDeleteJob.js

```diff
@@ -0,0 +1,49 @@
+const R = require('ramda')
+
+const Job = require('../../../../../job/job')
+
+const Survey = require('../../../../../../common/survey/survey')
+
+const RecordManager = require('../../../../record/manager/recordManager')
+const SurveyManager = require('../../../../survey/manager/surveyManager')
+
+class RecordsRemovedCyclesDeleteJob extends Job {
+
+  constructor (params) {
+    super(RecordsRemovedCyclesDeleteJob.type, params)
+  }
+
+  async execute (tx) {
+    this.total = 2
```

Comment:
what about ```this.total = 1 + cycleKeysDeleted.length```?

Reply:
uhm, the calculation of cycleKeysDeleted requires some time, and it's not even correct to add the entire lenght of cycleKeysDeleted to the total (the deletion is done with a single query)

Reply:
ok!